.travis.yml

```diff
@@ -14,32 +14,22 @@ services:
 
 env:
   matrix:
-  - NODE_VERSION=8 COUCH_VERSION=2
-  - NODE_VERSION=6.10 COUCH_VERSION=1
+  - NODE_VERSION=8
+  - NODE_VERSION=6.10
```

Comment:
We're also dropping node 6 support with 3.0.0 because 8.10 is LTS too. This means we could either drop down to 1 job on CouchDB 2 and Node 8, _or_ if you like keep the two jobs and test Node 8 and Node 9. What do you think?

Reply:
I like us testing against our minimum and also looking ahead to catch new problems if they arise, so I think both is best.

I'll change it to 8 and 9, and also change api / sentinel start warnings to those minimums as well.